src/components/community/Levels/components/Level/hooks/useLevelAccess.ts

```diff
@@ -4,6 +4,27 @@ import useBalance from "hooks/useBalance"
 import type { AccessRequirement } from "temporaryData/types"
 import useNeededAmount from "../../../hooks/useNeededAmount"
 
+const hasAccessToLevel = (
```

Comment:
logic could be moved back into the hook, there's no point to have it in a separate function anymore, or is there?